pkg/operator/genericoperatorclient/dynamic_operator_client.go

```diff
@@ -170,6 +174,54 @@ func (c dynamicOperatorClient) UpdateOperatorStatus(ctx context.Context, resourc
 	return retStatus, nil
 }
 
+func (c dynamicOperatorClient) ApplyOperator(ctx context.Context, fieldManager string, applyConfiguration *applyoperatorv1.OperatorSpecApplyConfiguration) (err error) {
+	applyMap, err := runtime.DefaultUnstructuredConverter.ToUnstructured(applyConfiguration)
+	if err != nil {
+		return fmt.Errorf("failed to convert to unstructured: %w", err)
+	}
+	applyUnstructured := &unstructured.Unstructured{
+		Object: map[string]interface{}{
+			"spec": applyMap,
+		},
+	}
+	applyUnstructured.SetGroupVersionKind(c.kind)
+	applyUnstructured.SetName(c.configName)
+
+	_, err = c.client.Apply(ctx, c.configName, applyUnstructured, metav1.ApplyOptions{
+		Force:        true,
```

Comment:
I think this is important so we can take ownership of the field

Reply:
I think so, since the operator owns it's own resource.